recipes-kernel/linux/linux-nilrt-next_git.bb

```diff
@@ -1,6 +1,6 @@
 DESCRIPTION = "NILRT linux kernel next development build"
 
-require linux-nilrt-next.inc
+require linux-nilrt-extra.inc
```

Comment:
linux-nilrt-extra kind of sounds like it installs non-essential kernel components. What about linux-nilrt-alternate instead?

Reply:
Sounds good. I'll  change the name on the next patch rev.